src/clr-angular/data/datagrid/render/constants.ts

```diff
@@ -9,3 +9,5 @@
 export const NO_LAYOUT_CLASS = 'datagrid-no-layout';
 export const COMPUTE_WIDTH_CLASS = 'datagrid-computing-columns-width';
 export const STRICT_WIDTH_CLASS = 'datagrid-fixed-width';
+export const HIDDEN_COLUMN_CLASS = 'datagrid-column-hidden';
```

Comment:
I don't see any reason why we can't just have 1 hidden class for both columns and cells, that will make it easier later to dedupe some of the header/cell renderer logic.